### PR TITLE
Lebovits/issu643 add frontend formatting check

### PR DESCRIPTION
**Pull request by nlebovits**
Should close issue #643 to add a formatting check for the front end on PR.

### CODE REVIEW

File: .github/workflows/pr_checks.yaml [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+  lint:
     runs-on: ubuntu-latest
+
     steps:
-      - name: Check out repository code
-        uses: actions/checkout@v4
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
+      - name: Checkout repository
+        uses: actions/checkout@v3
```
Comment (CodeWritingCow) [inline .github/workflows/pr_checks.yaml:14]: Hi @nlebovits why is the PR downgrading `actions/checkout` from v4 to v3?

**Review by CodeWritingCow — COMMENTED** [review]
@nlebovits please see my question about checkout@v3 vs v4.

File: .github/workflows/pr_checks.yaml (hunk 1 above)
Comment (nlebovits) [reply .github/workflows/pr_checks.yaml:14]: @CodeWritingCow good catch! sloppy chatgpt hallucinations lol

Comment (nlebovits) [reply .github/workflows/pr_checks.yaml:14]: will fix in a new commit + check the other PR for backend work to do the same

Comment (nlebovits) [reply .github/workflows/pr_checks.yaml:14]: @CodeWritingCow going to add a `dependabot.yml` to keep these up to date. see #757

**Comment by CodeWritingCow** [comment]
Hi @nlebovits the PR has a merge conflict. Can you resolve it? Thanks!

**Pull request merged**